docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,88 @@
+
+Important question is how to achieve log rotations so that logs won't consume all available space on the node. Right now, logrotate tool solves this task. Detailed configuration can be found [there](https://github.com/kubernetes/kubernetes/blob/release-1.5/cluster/gce/gci/configure-helper.sh#L96). In short, up to `5` rotation are kept, rotation is performed daily or if log file grows beyond `10MB`. Rotations belong to a single container, that is if container dies several times or pod gets evicted, all rotations for the container are lost.
+
+When user performs [`kubectl logs`](/docs/user-guide/kubectl/kubectl_logs), as shown above, request ends up on the kubelet, which in turn reads directly from the log file and returns its content in the response. __Note! Only last rotation is returned, the rest should be extracted manually.__
+
+### System components logs
+
+The information above is only about application containers. There are several system components (for example, `kube-proxy`), which go around this mechanism and write directly to the files in `/var/log` directory in the host filesystem. These files are also rotated daily and based on size, but size retention for them is higher, `100MB` instead of `10MB`.
+
+System components use the [glog](https://godoc.org/github.com/golang/glog) logging library. Conventions for logging severity for those components are described in [docs/devel/logging.md](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/logging.md).
+
+## Cluster-level logging architectures
+
+### Using node logging agent
+
+![Using node level logging agent](/images/docs/user-guide/logging/logging-with-node-agent.png)
+
+Logging agent is a tool to expose or push logs to a backend. From high level perspective it's a container, which has access to a directory with log files from all containers. This container should be on every node, therefore it should be either manifest pod, shipped with a node, or a daemon set replica.
+
+This is the most common and encouraged approach, because only one logging agent per node is created and it doesn't require any changes in the target application. __Note! Currently, it only works for application's standard output and standard error!__
+
+#### Default solutions
+
+Kubernetes doesn't specify logging agent, but it ships with two default options. Both use fluentd with custom configuration as an agent, both are shipped as manifest pods.
```

Comment:
Kubernetes doesn't specify a logging agent, but it ships with two default options: [Stackdriver Logging](/docs/user-guide/logging/sdl) for use with Google Cloud Platform,  and [Elasticsearch](/docs/user-guide/logging/elasticsearch). Both use fluentd with custom configuration as an agent on the node, and both are shipped as manifest pods.